sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.getApplicationForCOE.e2e-spec.ts

```diff
@@ -189,6 +189,112 @@ describe("ConfirmationOfEnrollmentInstitutionsController(e2e)-getApplicationForC
       });
   });
 
+  it("Should discount previous tuition remittance amount when there is a previous tuition remittance requested amount.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(
+      appDataSource,
+      {
+        institution: collegeC,
+        institutionLocation: collegeCLocation,
+        disbursementValues: [
+          createFakeDisbursementValue(
+            DisbursementValueType.CanadaLoan,
+            "CSLF",
+            1000,
+          ),
+        ],
+        secondDisbursementValues: [
+          createFakeDisbursementValue(
+            DisbursementValueType.CanadaLoan,
+            "CSLF",
+            1000,
+          ),
+        ],
+      },
+      {
+        applicationStatus: ApplicationStatus.Enrolment,
+        createSecondDisbursement: true,
+        firstDisbursementInitialValues: {
+          tuitionRemittanceRequestedAmount: 300,
+        },
+      },
+    );
+    const [_, secondDisbursementSchedule] =
```

Comment:
Please remove the "_" to avoid the VC Code warning. There is no need to have a variable declared, it can be
`const [, secondDisbursementSchedule]`

![image](https://github.com/bcgov/SIMS/assets/61259237/32354770-21fe-4342-ab7b-8f316bb9e05c)